src/export/InstanceExporter.ts

```diff
@@ -142,7 +142,11 @@ export class InstanceExporter implements Fishable {
       let instanceChild = instance[`_${childPathEnd}`] ?? instance[childPathEnd];
       // If the element is a choice, we will fail to find it, we need to use the choice name
       if (instanceChild == null && childPathEnd.endsWith('[x]')) {
-        const choiceSlices = children.filter(c => c.path === child.path && c.sliceName);
+        const possibleChoiceSlices = children;
```

Comment:
This is pretty small, but when I was using the debugger trying to understand what was going on, I noticed that `children` was changing as new items were pushed to `possibleChoiceSlices`. It doesn't seem like there are any unintended consequences to this, but would it potentially be safer to use a `cloneDeep` here? What do you think?

Reply:
I agree w/ Julia... if the intent is that `children` _should_ change, then I'd rather just remove line 145 and refer to `children` in the logic that follows.  If the intent is that `children` should not change, then `possibleChoiceSlices` should be a different array.

If we're not mutating the _elements_ themselves (but rather, just the _array_), then `children.slice()` would probably be more efficient than `cloneDeep(children)`.